faktorio-fe/src/pages/invoice/ContactComboBox.tsx

```diff
@@ -33,11 +33,24 @@ export function ContactComboBox(props: {
   const { value } = props
 
   useEffect(() => {
-    if (!value) {
+    const defaultContactValue = contacts.length === 1 ? contacts[0].id : ''
+    if (!value && lastInvoice && contacts.length > 0) {
       const lastUsedContactId = lastInvoice?.client_contact_id
-      if (lastUsedContactId) {
-        props.onChange?.(lastUsedContactId) // preselect the last used contact
+
+      const firstContact = contacts[0]
+
+      // NOTE: when the newest contact is added recently, select it
+      if (
```

Comment:
this was kinda despreate an parameter in the ULR would be vetter, i guess, but it works. It might cause trobules though...